web_timeline_calendar/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Web Timeline Calendar',
+    'summary': """
+        Timeline view for calendar""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
+    'depends': [
+        'calendar',
+        'web_timeline',
+        'web_widget_color',
+    ],
+    'data': [
+        'views/calendar_event.xml',
+        'security/ir.model.access.csv',
+        'data/calendar.event.color.csv',
```

Comment:
Is this not demo data?

Reply:
It is not demo data but it is Customer specific so the best could to add them manually on the customer environment or add them into a customer module. I'll check that with Souheil

Reply:
We are going to put them as "demo" for the OCA module. It will be added as data in a customer' specific module.